microbench/src/main/java/com/twitter/hpack/microbench/HeadersSize.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.hpack.microbench;
+
+import java.io.*;
+import java.util.List;
+
+/**
+ * Enum that indicates the size of the headers to be used for the benchmark.
+ */
+public enum HeadersSize {
```

Comment:
It has to be public for use with JMH.  I can make the methods public for consistency.